TemporalAirlinesConcept.Services/Implementations/Purchase/PurchaseWorkflow.cs

```diff
@@ -0,0 +1,262 @@
+﻿using TemporalAirlinesConcept.Common.Helpers;
+using TemporalAirlinesConcept.Services.Models.Flight;
+using TemporalAirlinesConcept.Services.Models.Purchase;
+using Temporalio.Common;
+using Temporalio.Exceptions;
+using Temporalio.Workflows;
+
+namespace TemporalAirlinesConcept.Services.Implementations.Purchase;
+
+[Workflow]
+public class PurchaseWorkflow
+{
+    private Saga _saga = new([]);
+
+    private string _flightId;
+
+    private bool _isSeatsReserved;
+
+    private bool _isPaid;
+
+    private bool _isCancelled;
+
+    private readonly ActivityOptions _activityOptions = new()
+    {
+        StartToCloseTimeout = TimeSpan.FromSeconds(60),
+        RetryPolicy = new RetryPolicy
+        {
+            InitialInterval = TimeSpan.FromSeconds(15),
```

Comment:
Usually when altering activities to change from the default, it's not a global change but specific to an activity